java/recursion/NCR.java

```diff
@@ -0,0 +1,42 @@
+
+
+import java.util.Scanner;
+
+/**
+ * Created by buddhimah on 10/16/2017.
+ */
+public class NCR {
+    public static  int fac(int num){
+
+        int r;
+
+        if(num==1){
```

Comment:
Handle negative cases.

Reply:
Thank you I will handle it

Reply:
Its Done :)

Reply:
@utkarsh-shekhar The change request you asked for completed